app/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Step 0 - Build
+FROM tiangolo/node-frontend:10 AS build
+
+# If REACT_APP_HTTP_URI variable is not passed as argument during image build,
+# it defaults to "http://localhost:4000"
+# i.e. docker build --build-arg REACT_APP_HTTP_URI=${ANY REACT_APP_HTTP_URI} ...
+ARG REACT_APP_HTTP_URI="http://localhost:4000"
+ENV REACT_APP_HTTP_URI=$REACT_APP_HTTP_URI
```

Comment:
It probably doesn't make sense to treat this as a build arg, so you can just omit the `ARG` line here and set the default value directly on the `ENV` line